client/tests/simulator_tests.rs

```diff
@@ -83,6 +83,63 @@ fn test_get_capability_manufacturer_id() {
     assert_eq!(received, expected);
 }
 
+#[test]
+fn test_get_random() {
```

Comment:
Probably worth having a test that checks that if you ask for a size larger than `Tpm2bDigest` that it cuts back the response properly and doesn't fail or panic.

Reply:
Fixed 👍